src/main/scala/PcapExample.scala

```diff
@@ -69,14 +63,16 @@ object PcapExample {
         !(data + offsetBytes + n)
       }
       .foreach { v =>
-        native.stdio.printf(toCString("%02X"), v)
+        native.stdio.printf(c"%02X", v)
       }
     println("...]")
   }
 
-  def main(args: Array[String]): Unit = {
+  def main(args: Array[String]): Unit = Zone { implicit zone => run(args) }
```

Comment:
The Zone is actually for the toCString, which requires an implicit allocation Zone.  Using the c interpolator instead of toCString(constant) avoids requiring a Zone, but that doesn't work if you have a dynamic string.